dataset/src/main/scala/frameless/ops/AggregateTypes.scala

```diff
@@ -0,0 +1,19 @@
+package frameless
+package ops
+
+import shapeless._
+
+trait AggregateTypes[V, U <: HList] {
```

Comment:
Could not make it work with `TypedAggregate[V, ?]` directly (which is another syntax for `Lambda[X => TypedAggregate[T, X]]` in kind projector), the compiler does not seems to be able to resolve the `Comapped` without the type alias trick I did in the linked commit :(